### DISCUSSION TITLE
Identification of individual areas in the main window

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#249] [General]
When checking the German translation in the "Layout" menu, I noticed that some texts there are not translated quite understandable.

In the main window there is a "QSO Table" at the top and a "New QSO" area below. In the program and in the Wiki this area is always called "New QSO". Visually, the two areas are not separated by a thin line or a note "New QSO". So, or something like that.

![Screenshot_20231021_112203](https://github.com/foldynl/QLog/assets/26311402/5596f0c3-9a13-4957-9fa6-c584cafc2913)

In the layout editor I noticed that in the selection menu below the texts "New QSO Rows" and "New QSO Detail Columns" are used.

![Screenshot_20231021_111750](https://github.com/foldynl/QLog/assets/26311402/8fd838ef-181d-4637-a64c-15b9984e65fd)

The text can be understood that these are new lines for a QSO. However, these are not new QSO lines, but an existing line in the "New QSO" area that can be adjusted.

One could perhaps use "'New QSO' Row" here, or think once generally, how one can delimit the individual ranges of the main window and/or designate more exactly.

---

This is just an initial thought of mine, though at the moment I don't know if this is something to think about further either.

**Comment by foldynl** [comment]
Wolfgang, you are right. When I implemented this, I didn't know exactly what to call it. I had several variants, but all were worse than the current text. Actually, I don't know exactly what to call it so that it is understandable.

It is necessary to find a short text that describes it better.

I leave it like that, but I'm really open to any improvements, because I wrote, I don't really like it.
